packages/lesswrong/components/posts/PostsTitle.tsx

```diff
@@ -121,7 +121,8 @@ const PostsTitle = ({post, postLink, classes, sticky, read, showQuestionTag=true
   const shared = post.draft && (post.userId !== currentUser?._id) && post.shareWithUsers
 
   // const shouldRenderQuestionTag = (pathname !== "/questions") && showQuestionTag
-  const shouldRenderEventsTag = pathname !== communityPath;
+  const shouldRenderEventsTag = (pathname !== communityPath) && (pathname !== '/pastEvents') && (pathname !== '/upcomingEvents') &&
+    !pathname.includes('/events') && !pathname.includes('/groups');
```

Comment:
I snuck one more thing in this PR: hiding the `[Event]` tag in the post title when we are in the Community section of the forum, so that [this kind of thing](https://forum.effectivealtruism.org/groups/YeW2gwh4gHexYQBjs) doesn't happen.

Reply:
Not beautiful, but seems fine.